tools/ci_import

```diff
@@ -0,0 +1,99 @@
+#!/bin/env python
+
+import argparse
+import json
+import os
+import sys
+import zipfile
+import tempfile
+from os import path
+
+import gitlab
+
+IMI_FOLDER = "generated-image-infos"
+MANIFEST_DB_PROJECT_ID = 36844106
+
+
+def import_image_infos(pipeline, token, verbose=False):
+    success = True
+    gl = gitlab.Gitlab("https://gitlab.com", oauth_token=token)
+    project = gl.projects.get(MANIFEST_DB_PROJECT_ID)
+    pipeline = project.pipelines.get(pipeline)
+    jobs = pipeline.jobs.list(all=True)
+    for job in jobs:
+        if verbose:
+            print(f"Job {job.get_id()}:")
+        directory = path.join(tempfile.TemporaryDirectory().name, str(job.get_id()))
```

Comment:
We usually try using `tempfile.TemporaryDirectory` as a context manager, so it gets cleaned up predictably. I think we can just use one tempdir for the whole `import_image_infos`.

See e.g. https://github.com/osbuild/osbuild-composer/blob/df22e8b6869b32018a10eb812bc37ff8878c3ab5/tools/deploy-qemu#L36 (there is also a ton of examples in `osbuild`)